data/fits/fits.txt

```diff
@@ -0,0 +1 @@
+Polyphy may crash if this folder doesn't exist!
```

Comment:
The best way to handle it is to use Python to check if the path exists else create it.

Reply:
ive squashed the commits and made this fix for all the jupyter notebooks + removed fits folder - let me know if there's anything else I can do here, thanks!

Reply:
Looks good to me. Thank you for the addition @deehrlic .